forem/forem
# Add user status from profiles to the report section

## What type of PR is this? (check all applicable)


## Description

Add user status from profiles to the report section. For more details follow below sections

## Related Tickets & Documents



- Closes #19211

## QA Instructions, Screenshots, Recordings

[Figma link](https://www.figma.com/file/9UfSjNYJs745w3Q8JCUqsL/Small-Wins---%E2%80%A8Core-Experience-%E2%9A%A1%EF%B8%8F?node-id=1500%3A5744)

| Before  | After |
| ------------- | ------------- |
| <img width="1298" alt="Screenshot 2023-03-20 at 3 06 58 PM" src="https://user-images.githubusercontent.com/9396084/226301024-47c54b2b-96a0-4b89-a60b-0baf2c1e7502.png"> | <img width="1273" alt="Screenshot 2023-03-20 at 3 06 40 PM" src="https://user-images.githubusercontent.com/9396084/226687277-2ed2f321-09a5-4cdb-b564-429184e45ffc.png"> |

**How to test?**
1. Sign in as admin locally
2. Visit any reports like http://localhost:3000/admin/moderation/reports/2 or http://localhost:3000/admin/moderation/reports/1 or http://localhost:3000/admin/moderation/reports/3
3. Notice in each of these reports there is a status-label like `Trusted`, `Good standing`, etc next to the user name.

### UI accessibility concerns?

Improves admin accessibility 

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [x] This change does not need to be communicated
